dashboard/app/views/levels/editors/_collapsed.html.haml

```diff
@@ -0,0 +1,9 @@
+- xml_id = "#{xml_id}"
+- collapse_xml = "#{xml_id}_collapse_xml"
+
+.min_max_label{data: {toggle: "collapse", target: "##{collapse_xml}"}}
+  Open/Close
+.row.collapse.in{id: collapse_xml}
```

Comment:
nit: are either the `.in` or the `.min_max_label` classes doing anything?

Reply:
`.in` is to make the dropdown open by default (Asked for by Kiki)
`.min_max_label` was just to have something for the data  toggle to have something to hang off of.